e2e/test-component/scenarios/embedding-sdk/interactive-question-theming.cy.spec.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  InteractiveQuestion,
+  type MetabaseTheme,
+} from "@metabase/embedding-sdk-react";
+
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import { createQuestion, describeEE } from "e2e/support/helpers";
+import {
+  mockAuthProviderAndJwtSignIn,
+  mountSdkContent,
+  signInAsAdminAndEnableEmbeddingSdk,
+} from "e2e/support/helpers/component-testing-sdk";
+import { getSdkRoot } from "e2e/support/helpers/e2e-embedding-sdk-helpers";
+import { Box } from "metabase/ui";
+
+const { ORDERS, ORDERS_ID } = SAMPLE_DATABASE;
+
+describeEE(
+  "scenarios > embedding-sdk > interactive-question > theming",
+  // realHover color check is flaky 10% of the time so a retry is added
+  { retries: { runMode: 2, openMode: 2 } },
```

Comment:
I tried using `getComputedStyle(element, :hover)` instead but that doesn't work as it requires the exact selector the `:hover` is registered on, which is troublesome. Using `realHover` is more realistic but flakes 5 - 10% of the time, so an additional retry helps in that case